saleor/wombat/management/commands/create_wombat_user.py

```diff
@@ -0,0 +1,18 @@
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.contrib.auth import get_user_model
+
+User = get_user_model()
+WOMBAT_USERNAME = getattr(settings, 'WOMBAT_USERNAME', 'wombat')
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
```

Comment:
Do we need command for creating only one user with hard coded name? I propose to create data migration for it.